src/commands/session.ts

```
@@ -45,12 +49,12 @@
 		const totals = calculateTotals(sessionData);
 
 		// Show debug information if requested
-		if (ctx.values.debug && !ctx.values.json) {
+		if (ctx.values.debug && !useJson) {
```

ESLint

[error] Unexpected any value in conditional. An explicit comparison or type conversion is required.

(ts/strict-boolean-expressions)